library/src/main/kotlin/dev/jahir/frames/ui/fragments/SettingsFragment.kt

```diff
@@ -84,7 +84,7 @@ open class SettingsFragment : BasePreferenceFragment() {
 
         val downloadLocationPref = findPreference<Preference?>("download_location")
         downloadLocationPref?.summary =
-            string(R.string.download_location_summary) + "\n${preferences.downloadsFolder}"
+            "${preferences.downloadsFolder}"
```

Comment:
this can be in the same line (86)
```suggestion
            preferences.downloadsFolder
```